mlflow/environment_variables.py

```diff
@@ -581,7 +581,9 @@ def get(self):
 
 # Whether or not to enable trace logging in MLflow in served models.
 # TODO: Document the behavior of this flag.
-MLFLOW_ENABLE_TRACE_IN_SERVING = _BooleanEnvironmentVariable("MLFLOW_ENABLE_TRACE_IN_SERVING", True)
+MLFLOW_ENABLE_TRACE_IN_SERVING = _BooleanEnvironmentVariable(
```

Comment:
nit: could you add some comment for the default decision, based on our offline discussion?

Reply:
@B-Step62 sounds good. I'll add a comment. Just to confirm, the default value here must align with `mlflowserving`'s default, right?

Reply:
> Just to confirm, the default value here must align with mlflowserving's default, right?

Yes! Also setting `False` is safer in case the serving does not set the env var (e.g. user deploys the model with mlflow 2.13.1 in the workspace where the serving change is not ramped up yet).